src/app/views/templates/styles.ts

```diff
@@ -56,3 +56,69 @@ export const linkStyle: NonNullable<LinkProps['style']> = {
   wordBreak: 'break-all',
   color: '#445fcd',
 }
+
+const textStyles = {
```

Comment:
I'm thinking that we should separate MRF styles out from the basic plain styles.

MRF emails (sent to MOP) are fancier while our other emails (to admins to inform about various failures) are less fancy.

Reply:
seems a little odd to have a style sheet just for mrf emails. I'm thinking that when we release more emails in the future, the emails sent out should conform to the same style standard as the new mrf email forms to maintain consistent product quality to all users (regardless of MOP / admins).

Reply:
I get what you mean, but having `textStyles` and `textStyle` both in the same file brings more confusion than consistency if that's the play in the current moment.

Reply:
split it into `commonStyle` and `mrfCompletionEmailStyle` ts files 
- common is for reusable stuff that can be used across new email templates eg, body-1, h1, design system color hex codes 
- mrfCompletionEmailStyle is for specific styles for the email itself eg, container sizing, question and answer margins

Reply:
I'm thinking in the future, if more templates were to be added: 
- import the common styles from `commonStyle.ts` 
- create component specific styles in the new `..specificComponentStyle.ts` class

Reply:
Sounds good! Likely this can also follow figma -- don't think we have figma versions of it for our older emails, and we're just starting to add chromatic.